symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/model/AttachmentType.java

```diff
@@ -0,0 +1,41 @@
+package com.symphony.bdk.core.service.message.model;
+
+import org.apiguardian.api.API;
+
+@API(status = API.Status.EXPERIMENTAL)
+public enum AttachmentType {
```

Comment:
This list is not fixed : https://developers.symphony.com/restapi/reference#attachment-types :) Therefore I'm not sure that we can really provide an `enum` here

Reply:
I'm also wondering about this one. But I really don't like to make another API call to just create a message.

Reply:
or maybe we can let the user specify the whatever extension file type they want?

Reply:
> I'm also wondering about this one. But I really don't like to make another API call to just create a message.

Of course we won't make this call for each message creation. However, I think it is also import to adopt a "fail-fast" strategy directly at BDK level, this would avoid overloading Agent/SBE with requests that will in the end fail because the file type is not allowed. 

I see 2 strategies that we could implement: 
- lazy load the list of allowed attachments during first call to send message
- load this list at BDK startup (both in `SymphonyBdk` and in Spring Boot starter)